src/toil/test/cwl/cwlTest.py

```diff
@@ -153,46 +153,22 @@ def test_run_conformance(self, batchSystem=None):
             shutil.move("common-workflow-language-%s" % testhash, cwlSpec)
             os.remove("spec.zip")
         try:
-            cmd = ["bash", "run_test.sh", "RUNNER=toil-cwl-runner",
-                   "DRAFT=v1.0", "-j4"]
+            cmd = ['bash', 'run_test.sh', 'RUNNER=toil-cwl-runner', 'DRAFT=v1.0',
+                   'TEST_N="--timeout=1800"', '-j4']
```

Comment:
Nice hack (ab)using `TEST_N`, but `--timeout` is for `cwltest`, so just call that directly:

```
cd v1.0
cwltest --tool ${path_to_toil-cwl-runner} --test=conformance_test_v1.0.yaml --timeout=1800 --basedir=${PWD}
```

Reply:
Yes, that's much cleaner, and I'll make the change. I felt odd about it so thanks for replying back.

I was hoping to push back a change to the bash script to include another var to adjust the timeout if you feel like that would be useful, but that would be for the next hash.